pkg/status/README.md

```diff
@@ -0,0 +1,12 @@
+# pkg/status
+
```

Comment:
```suggestion
<!-- status autogenerated section -->
<!-- end autogenerated section -->
```
We should include an autogenerated section in here that will be filled out using the information from `metadata.yaml` when `make -C pkg/status generate` is run.